Dockerfile

```diff
@@ -2,19 +2,28 @@
 FROM python:3.6
 
 # Update system
-RUN apt-get update && apt-get -y upgrade && apt-get -y autoremove
+RUN apt-get update && \
+    apt-get -y upgrade && \
+    apt-get -y autoremove
 
 # Install system dependencies
-RUN apt-get install -y build-essential sqlite3
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+                    build-essential \
+                    sqlite3
 
 # Create main code folder
 RUN mkdir /code
 WORKDIR /code
 
-# Install dev dependencies (simulates `make install`)
-COPY ./requirements-dev.txt requirements-dev.txt
+# Install dependencies (simulates `make install` in the live dockerfile)
 RUN pip install -U pip==19.3.1
 RUN pip install -U pip-tools
-RUN pip-sync requirements-dev.txt
+COPY ./requirements.txt requirements.txt
```

Comment:
why not
```
COPY ./requirements.txt requirements.txt
COPY ./requirements-dev.txt requirements-dev.txt
RUN pip-sync requirements.txt requirements-dev.txt
```
?

Reply:
Was trying to not invalidate the cache of the container build when only the dev requirements were updated.

Also, if you pass 2 requirements files to `pip-sync` how does it handle them? If it does them one by one, wouldn't it delete the dependencies installed by the first one?

Reply:
That's what the docs recommend for layered requirements in https://github.com/jazzband/pip-tools#workflow-for-layered-requirements. 

Since running pip-compile right now will update the package versions and likely update the prod requirements, I don't think it would hurt to have installing both in a single step. Not a blocker though, I'm happy for it to be merged as is.

Reply:
You are right I will do it the recommended way. Thanks for the tip 👍